code/controllers/master.dm

```diff
@@ -17,48 +17,61 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 /datum/controller/master
 	name = "Master"
 
-	// Are we processing (higher values increase the processing delay by n ticks)
-	var/processing = TRUE
```

Comment:
This is what happens when people change TRUE/FALSE willy nilly without actually thinking.